service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -324,7 +322,7 @@ public void testGetWithResetForResourcesByIds() throws Exception {
         assertEquals(cachedResult.getEffect(), result.getEffect());
 
         Thread.sleep(1);
-        this.cache.resetForResourcesByIds(ZONE_NAME, new HashSet<>(Arrays.asList(XFILES_ID)));
+        this.cache.resetForResourcesByIds(ZONE_NAME, new HashSet<>(Collections.singletonList(XFILES_ID)));
```

Comment:
Use `Collections.singleton(..)` instead of `new HashSet<>(Collections.singletonList(..))`